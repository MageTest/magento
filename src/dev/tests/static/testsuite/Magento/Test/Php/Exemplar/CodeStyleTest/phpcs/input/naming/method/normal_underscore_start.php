<?php
class Magento_Test_Php_Exemplar_CodeStyleTest_phpcs_input_naming_method_normal_underscore_start {
    private function _testFunctionPrivate()
    {}

    protected function _testFunctionProtected()
    {}
}
