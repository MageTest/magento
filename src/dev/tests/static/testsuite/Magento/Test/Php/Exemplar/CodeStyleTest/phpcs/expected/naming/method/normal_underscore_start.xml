<?xml version="1.0" encoding="UTF-8"?>
<config>
    <incomplete/>
    <total errors="0" warnings="0"/>
</config>
